Ushindi-Zirimwabagabo-Josue/Gravity-House
# #172911021 create the project

#### What does this PR do?
create the project

#### Description of Task to be completed?
-create a virtual environment
-install Django

#### How should this be manually tested?
For the virtual environment
-open vs code
-open the terminal by pressing **ctr+shift+`**
-in the terminal, type "**venv\Scripts\activate.bat**"
For Django
-Make sure you are in your virtual environment
-type "**pip freeze**"

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#172911021]https://www.pivotaltracker.com/story/show/172911021

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
